code/modules/antagonists/heretic/items/keyring.dm

```diff
@@ -185,7 +185,7 @@
 
 	if(reference_resolved)
 		make_portal(user, reference_resolved, target)
-		to_chat(user, span_notice("You use [src], to link [link] and [target] together."))
+		to_chat(user, span_notice("You use [src], to link \the [reference_resolved] and \the [target] together."))
```

Comment:
For the record the `\the` helpers here are doing absolutely nothing, `\the` is added regardless when you stringify something under most circumstances

Reply:
The airlocks in runtime station weren't adding it

Reply:
It adds it if necessary, so in this case it did not deem it necessary

Reply:
```suggestion
		to_chat(user, span_notice("You use [src], to link [reference_resolved] and [target] together."))
```